src/controller/tests/BUILD.gn

```diff
@@ -1,31 +1,29 @@
-# Copyright (c) 2020 Project CHIP Authors
+#Copyright(c) 2020 Project CHIP Authors
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+#Licensed under the Apache License, Version 2.0(the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
 #
-# http://www.apache.org/licenses/LICENSE-2.0
+#http: // www.apache.org/licenses/LICENSE-2.0
 #
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
 
 import("//build_overrides/build.gni")
 import("//build_overrides/chip.gni")
 import("//build_overrides/pigweed.gni")
+import("${chip_root}/build/chip/chip_test_suite.gni")
 import("${chip_root}/src/controller/flags.gni")
 import("${chip_root}/src/lib/lib.gni")
-
-import("${chip_root}/build/chip/chip_test_suite.gni")
-
 chip_test_suite("tests") {
   output_name = "libControllerTests"
 
   test_sources = []
 
-  # Not supported on efr32.
+  #Not supported on efr32.
```

Comment:
Same here: it looks like some linter removed the first whitespace inside comments. Please undo that throughout